### DISCUSSION TITLE
Looking for ways to ignore SLO's that already exist.

### DISCUSSION THREAD

**Discussion by jjesky** [Dynatrace/dynatrace-configuration-as-code#724] [General]
I am trying to deploy configuration changes to dynatrace and getting failures when the SLO already exists and there is no PUT operator for this API. How can I get by this? For example I am adding configurations and run the pipeline, the SLO is created, I rerun and it fails saying it already exists.

**Comment by UnseenWizzard** [comment]
Hi @jjesky, do you have a minimal configuration sample in which you can reproduce this problem? 
And with which version of monaco and against what version of Dynatrace are you encountering this? 

In general SLOs are possible to be updated. 
Quickly checking the latest versions end-to-end test which among other configs creates, then updates an SLO config this also seems to work in 1.8.3.


> there is no PUT operator for this API

There is a `PUT /slo/{id} ` - as asked above, what version of Dynatrace are you seeing this/where did you look for the API?
